python/smashbox/utilities/__init__.py

```diff
@@ -293,6 +294,12 @@ def create_owncloud_group(group_name):
     oc_api.login(config.oc_admin_user, config.oc_admin_password)
     oc_api.create_group(group_name)
 
+def get_conflict_files(d):
+    conflict_files = []
+    conflict_files.extend(glob.glob(os.path.join(d,'*_conflict-*-*'))) # earlier versions use _conflic
```

Comment:
Now we support both versions

Reply:
nice - THX - I leave this to kuba for final review and test.